modules/carto/src/api/maps-v3-client.js

```diff
@@ -167,6 +183,70 @@ export async function getData({type, source, connection, credentials, geoColumn,
   }
 
   const {accessToken} = localCreds;
+  return {url, mapFormat, accessToken};
+}
 
-  return await request({url, format: mapFormat, accessToken});
+/* global setInterval, URLSearchParams */
+async function _getMapDataset(dataset, accessToken) {
+  // First fetch metadata
+  const {connectionName: connection, source, type} = dataset;
+  const {url, mapFormat} = await _getDataUrl({
+    credentials: {accessToken},
+    connection,
+    source,
+    type
+  });
+
+  // Extract the last time the data changed
+  const cache = parseInt(new URLSearchParams(url).get('cache'), 10);
```

Comment:
Do we really need this cache system? Data will be cached by the CDN and the browser.

If the data changes the URL will be different, if the data doesn't change the URL will be the same but with a cache-control of 1 year:

`cache-control: public,max-age=31536000,must-revalidate`

So I think we can use the default cache system of the browser and remove these lines

Reply:
The reason to grab the cache parameter is to avoid doing the second request at all. It is true that if the cache is unchanged the browser will return the response from the local cache, but the response will still get processed, which means the deck.gl layers will all get recreated